models/src/main/java/com/vimeo/networking2/ProgressiveVideoFile.kt

```diff
@@ -61,7 +61,7 @@ data class ProgressiveVideoFile(
      * The file size of the video.
      */
     @Json(name = "size")
-    val size: Int? = null,
+    val size: Long? = null,
```

Comment:
Optional, you could add kdocs to the params in this class that are missing them.

Reply:
The only params that I see without docs are overridden properties, which we do not document since they inherit the docs from the interface they are overriding.